cmd/internal/register/maps.go

```diff
@@ -99,12 +105,24 @@ func Maps(a *atlas.Atlas, maps []config.Map, providers map[string]provider.Tiler
 
 			// lookup our proivder
 			provider, ok := providers[providerLayer[0]]
+			fmt.Println("printing provider:", provider)
+
+			// determine if layers should automatically be created from provider regex
+			auto := false
+			for _, p := range conf.Providers {
+				if p["name"] == providerLayer[0] && p["auto"] == true {
+					auto = true
```

Comment:
I thought we got rid of the `auto`. I don't think we need it. I think this is unnecessary. The maps sections should not care if the Provider is auto-configured or not. It should just care about the layers it needs and it can do this by looking at the layer name to see if it returns multi-layers.

Reply:
@ARolek Mentioned that he wanted it to automatically configure even if there is no auto_config present. This requires an auto tag. We can't have both